packages/redux/src/categories/selectors/categories.ts

```diff
@@ -58,3 +60,37 @@ export const areCategoriesLoading = (state: StoreState) =>
  */
 export const areCategoriesFetched = (state: StoreState) =>
   getIsFetched(state.categories as CategoriesState);
+
+/**
+ * Returns a specific category by its id.
+ *
+ * @param state      - Application state.
+ * @param categoryId - Category id.
+ *
+ * @returns Category normalized or undefined is nothing found.
+ */
+export const getCategory = (
+  state: StoreState,
+  categoryId: Category['id'],
+): Category | undefined => getEntityById(state, 'categories', categoryId);
```

Comment:
Same here. Check for the rest of your changes also

Reply:
Reviewed typing for all the selectors on this PR. Can you take a look? Thanks!